include/inspsocket.h

```diff
@@ -227,6 +227,11 @@ class CoreExport StreamSocket : public EventHandler
 	};
 
  private:
+	/** Whether this socket should close once its sendq is empty */
+	bool closeonempty;
+
+	bool closing;
```

Comment:
Can you document this?

Reply:
Fixed in 1cec315078077b55d1417661954a54cee97232c8